test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -132,6 +132,19 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
         [InlineData("{'properties': {'key': {'type': 'string'}}}", "{'key': 1}",
             "['warning','unexpected-type','Expect type 'String' but got 'Integer'','file',1,9]")]
 
+        // additional properties validation
+        // AdditionalProperty is enabled with explicit false(can be named additionalProperties or additionProperties)
+        [InlineData("{'properties': {'key': {'type': 'string'}}, 'additionalProperties': {}}", "{'key': 'value', 'key1': 'value1'}", "")]
```

Comment:
remove line 18 `"additionalProperties"` to test against the spec

Reply:
@yufeih  there's an extra `patternProperties` which use regex to match keys, that we don't support yet. and it's not used in docs-ui.